backend/pom.xml

```diff
@@ -1,97 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Accidental deletion?

Reply:
Had moved it so that I could call maven in IntelliJ without reconfiguring due to me not knowing how

Reply:
But your teammates are using this file in its current location. It can move if doing so is required technically or is better organizationally but I wouldn't move it just to make intelliJ happy. If there's no way to configure intelliJ to use the file in its current location then there's always the `mvn` command line interface. WDYT?

FYI @jzwang8

Reply:
Agreed, I'm using the backend/pom.xml file and will push changes to it once my PR gets approved, so I think it would be best to keep this file!

Reply:
Yes, that is why is reverted the deletion after

Reply:
The MVN command interface wouldn't work due to here not being a POM.xml in the directory in which my project is structured and I haven't fully configured intellij on my corporate laptop . The mvn cli is the first thing that i tried and whenever i would run it i would get `The goal you specified requires a project to execute but there is no POM in this directory (/home/grantjustice/IdeaProjects/Ad-lib). Please verify you invoked Maven from the correct directory`

Reply:
what if you cd into the backend directory and re-run the command?

Reply:
I get the same error

Reply:
Can you share the error?

Reply:
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (/home/grantjustice/IdeaProjects/Ad-lib). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException

Reply:
That still looks like it's being run from the root instead of the backend directory. Can you try

```
cd /home/grantjustice/IdeaProjects/Ad-lib/backend
mvn compile
```